.pre-commit-config.yaml

```diff
@@ -1,8 +1,9 @@
 repos:
--   repo: https://github.com/pre-commit/pre-commit-hooks
+  - repo: https://github.com/pre-commit/pre-commit-hooks
     rev: v4.1.0
     hooks:
-    -   id: check-json
-    -   id: check-yaml
-    -   id: end-of-file-fixer
-    -   id: trailing-whitespace
+      - id: check-json
+        exclude: tsconfig.json
+      - id: check-yaml
+      - id: end-of-file-fixer
+      - id: trailing-whitespace
```

Comment:
is prettier missing on purpose?

Reply:
I was thinking about removing pre-commit and letting prettier take care of everything

Reply:
[lint-staged](https://github.com/okonet/lint-staged) looks good

Reply:
Replaced with husky and lint-staged. Lost the hooks for end of lines and end of files for now, could be replaced by fitting node hooks at some point.